tools/vcli/bw/repl/repl.go

```diff
@@ -425,9 +425,10 @@ func runBQL(ctx context.Context, bql string, s storage.Store, chanSize, bulkSize
 		})
 		return nil, msg
 	}
+	nRows := res.NumRows()
```

Comment:
I thing the old way is actually better, there is less variables and res.NumRows is pretty self explanatory, maybe res is not but that is another fix.

Reply:
What I wanted here was just to not capture the entire result table `res` inside the tracing closure just to have access to its `NumRows` later. The argument here is the same as the one for passing a `g.ID()` (graph ID) to a local variable `gID` before sending it to the tracer: we do not want to capture the entire graph inside the closure just to get its string `ID` later.